wiki/Help_centre/Upgrading_to_lazer/de.md

```diff
@@ -146,11 +157,13 @@ Scores werden unter "Neuliche Plays" angezeigt, aber noch nicht in "Beste Perfor
 
 #### Wenn ich einen Score in lazer erreiche, wird es dafür dann Performance-Punkte geben?
 
-Für Scores werden die Performance-Punkte bereits berechnet (du kannst das in der Sektion "Neuliche Plays" deines Profils erkennen), aber sie tragen noch nicht zum Gesamtwert bei.
+Der Abschnitt der kürzlichen Spiele auf der Profilseite zeigt die in lazer erreichten Scores mit den berechneten Performance-Punkten an. Die Gesamtanzahl wird dargestellt, sobald man mit der Maus über die Gesamtzahl der regulären Performance-Punkte fährt.
```

Comment:
Not sure if I got this right here. The sentence reads a bit weird to me in the original 🤔